### PR TITLE
Fix #3667 Blank page is displayed if user navigates to a deleted team

### LINKED ISSUE #3667: Blank page is displayed if user navigates to a deleted team
**Affected module**
UI

**Describe the bug**
Blank page is displayed if user navigates to a deleted team

**To Reproduce**
- Navigate to teams from the settings tab
- Delete a team (eg : Cloud Infra)
- Edit the URL and enter the name of the deleted team
- Blank page is displayed

https://user-images.githubusercontent.com/67048554/160597177-6ad9046c-130f-49c4-8e70-808d858fdd5f.mov

**Expected behaviour**
Proper message should be displayed instead of a blank page

**Version:**
 - OpenMetadata version: 0.9.1

**Additional context**
NA

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #3667 
### Describe your changes :

I worked on issue #3667 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00, @vivekratnavel

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx [hunk 1]
```diff
@@ -32,6 +34,10 @@ jest.mock('../../auth-provider/AuthProvider', () => {
   };
 });
 
+jest.mock('@fortawesome/react-fontawesome', () => ({
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx:37]: Can we mock this module globally in Jest config? Do not need to mock it on every file then

**Review by darth-coder00 — DISMISSED** [review]
One change requested!

File: openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx:37]: @darth-coder00, Sure I will make the changes.

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
@Sachin-chaurasiya Can you commit this file as well `src/test/unit/mocks/fontawesome.mock.js`

**Review by darth-coder00 — APPROVED** [review]
LGTM!

File: openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx [hunk 2]
```diff
@@ -320,3 +322,57 @@ describe('Test Teams page', () => {
     expect(await findByText(container, /ManageTab/i)).toBeInTheDocument();
   });
 });
+
+describe('Test Teams page sad path', () => {
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx:326]: Nit
```suggestion
describe('Test Teams page should handle errors', () => {
```

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**